cpp/include/rmm/aligned.hpp

```diff
@@ -61,10 +58,7 @@ static constexpr std::size_t CUDA_ALLOCATION_ALIGNMENT{256};
  *
  * @return True if the alignment is valid, false otherwise.
  */
-[[nodiscard]] constexpr bool is_supported_alignment(std::size_t alignment) noexcept
-{
-  return is_pow2(alignment);
-}
+[[nodiscard]] RMM_EXPORT bool is_supported_alignment(std::size_t alignment) noexcept;
```

Comment:
FWIW, I believe `RMM_NAMESPACE` already contains the `RMM_EXPORT` macro, but it's not bad to be explicit. If we wanted to, we could then go back to just saying `namespace rmm` in this file.

We should, however, probably add `RMM_EXPORT` to the `static` symbols as well in that case.

Reply:
Thanks for the explanation! I was unsure about this. I will think some more about what solution seems the most natural. Related to the other comment below, I would like it if we could use either `RMM_NAMESPACE` or `namespace rmm` in both headers and implementation files -- but that is tricky with the need to export things.

What do you think of getting rid of `RMM_NAMESPACE` and using `namespace RMM_EXPORT rmm` in the headers and `namespace rmm` in the implementation files?

Reply:
Yes, for sure. That will likely be for a later pass. I haven't touched any "detail" headers yet.